projects/plugins/social/src/js/store/selectors/connections.js

```diff
@@ -0,0 +1,5 @@
+const connectionsSelectors = {
+	getConnections: state => state.connections ?? null,
```

Comment:
Are we trying to force this to `null` rather than `undefined`? There might be an advantage to defaulting to an empty array so we don't need to check for null elsewhere.

Reply:
No real reason, I'll change it to `[]` 👍🏼